hw5/task1.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+// 0..10
+
+let i = 10;
+
+while (i >+ 0) {
+    console.log(i);
+    i--;
+}
+
+// :)
+const emoji = ':)';
+let row = emoji;
+
+for (i = 1; i < 6; i++) {
```

Comment:
Explicit variable declaration
```suggestion
for (let i = 1; i < 6; i++) {
```